src/milind/Array.java

```diff
@@ -0,0 +1,78 @@
+/*Assignment - 12 : 7th Oct'2020 
+write a single program having different methods.
+ 
+a) To find Min number from given array
+b) To find Max number from given array
+c) To find Avg of all numbers from give array
+d) To display all numbers [0 to length]
+e) To display all numbers [length to 0]
+g) Method to find average of min and max number from array.*/
+
+package milind;
+
+public class Array {
+	
+	int minNumberFromArray(int[] array) {
+		int minNumber = array[0];
+		for(int index = 0; index < array.length; index++) {
+			if(minNumber > array[index]) {
+				minNumber = array[index];
+			}
+		}
+		return minNumber;
+	}
+	
+	int maxNumberFromArray(int[] array) {
+		int maxNumber = array[0];
+		for(int index = 0; index < array.length; index++) {
+			if(maxNumber < array[index]) {
+				maxNumber = array[index];
+			}
+		}
+		return maxNumber;
+	}
+	
+	double arrayAvg(int[] array) {
+		int sum = 0;
+		for(int index = 0; index < array.length; index++) {
+			sum = sum + array[index];
+		}
+		double avg = sum / array.length;
+		return avg;
+	}
+	
+	void printArray(int[] array) {
+		System.out.print("Array Printing : ");
+		for(int index = 0; index < array.length; index++) {
+			System.out.print(array[index] + ", ");
+		}
+		System.out.println("");
+	}
+	
+	void printReverseArray(int[] array) {
+		System.out.print("Reverse array printing : ");
+		for(int index = array.length-1; index >= 0 ; index--) {
+			System.out.print(array[index] + ", ");
+		}
+		System.out.println("");
+	}
+	
+	double averageOfMinAndMax(int minNumberFromArray, int maxNumberFromArray) {

```

Comment:
I think we have requirement where you need to find max & min number then you need to find avg of both, so this expecting two variable in method looks wrong as per the implementation perspective, user will give us array, not min & max value. I hope you got my point.


You can write code like this, I remove variable and directly doing sum of min & max, no problem, if you are not comfortable to write code like below, just optimize.

int avgOfMinMax = (getMinNumber(input)+getMaxNumber(input))/2